src/modules/Tasklist.test.js

```diff
@@ -0,0 +1,80 @@
+const Tasklist = require('./Tasklist.js');
+
+const tasklist1 = new Tasklist();
+
+const deleteitem = (item) => {
+  const thisindex = parseInt(item.parentNode.id.substring(2), 10);
+  tasklist1.tasks.splice(thisindex, 1);
+  for (let i = thisindex; i < tasklist1.tasks.length; i += 1) {
+    tasklist1.tasks[i].index -= 1;
+  }
+  tasklist1.saveTasksToLocalStorage();
+  tasklist1.clearlist();
+  tasklist1.renderTasks();
+};
```

Comment:
- [ ] Please remove this function and use the function in the TaskList class and if you don't how to fire the click event in the javascript check this [link](https://www.w3schools.com/jsref/met_html_click.asp) it will help you.

Reply:
Done, thanks for the link